android/src/toga_android/libs/utilfile.py

```diff
@@ -0,0 +1,58 @@
+import abc
+import io
+
+import java
+
+BufferedReader = java.jclass("java.io.BufferedReader")
+InputStreamReader = java.jclass("java.io.InputStreamReader")
+Objects = java.jclass("java.util.Objects")
```

Comment:
You should be able to use `from java.io import BufferedReader` etc here.

Reply:
Thank you for the information